### DISCUSSION TITLE
[Q&A] 如何创建仅自己可用的agent呢？

### DISCUSSION THREAD

**Discussion by deku0818** [lobehub/lobehub#713] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

如何创建仅自己可用的agent呢？

[a comment by an automated account is not shown]

**Comment by deku0818** [comment]
我的目的是像GPTS一样，我想在团队内部分享工具。

[a comment by an automated account is not shown]

**Comment by canisminor1990** [comment, marked as answer]
不确定如何理解该问题：
- 如果指代个人创建助手存储，目前都是存在浏览器本地数据库中，你可以理解为就是你私人可用的 Agent
- 如果指代你想创建私人 Agent Market 并分享给团队使用 ，访问 https://github.com/lobehub/lobe-chat-agents 查看如何自部署私人 Agent Market

[a comment by an automated account is not shown]

**Comment by ilovefreesw** [comment]
I did that but now getting error. Can't install or use any agent.

https://github.com/lobehub/lobe-chat/discussions/4705
